ecl/hql/hqlgram2.cpp

```diff
@@ -10188,6 +10188,9 @@ void HqlGram::cloneInheritedAttributes(IHqlScope * scope, const attribute & errp
             curBase->getSymbols(syms);
             syms.sort(compareSymbolsByName);
 
+            QuickExpressionReplacer virtualMapper;
```

Comment:
Instead of creating an instance of this class each time quickFullReplaceExpression is called, use the same instance for all module members.  This will avoid re-transforming any code that the members have in common.